### PR TITLE
Iosevka font

### PR DESCRIPTION
**Pull request by depatchedmode**
Added the Iosevka web font

### CODE REVIEW

**Comment by shekhar-shubhendu** [comment]
@depatchedmode this PR seems to be doing more than just fonts

File: poll-taquito/package.json [hunk 1]
```diff
@@ -21,13 +21,15 @@
     "@types/node": "^12.0.0",
```
Comment (shekhar-shubhendu) [inline poll-taquito/package.json:21]: add this to build
```
"babel-loader": "8.1.0"
```

File: poll-taquito/src/components/Nav.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import Button from "@material-ui/core/Button";
+import { Link } from "react-router-dom";
+
+export default function Nav() {
+  // const { ... } = useParams();
+  const { disconnect, activeAccount, connect } = useWallet();
+  return (
+    <AppBar position="static">
+      <Toolbar>
+        <Link to="/" style={{flex:1,color:'white',fontWeight:'bold',textDecoration:'none'}}>HICVOTE</Link>
```
Comment (shekhar-shubhendu) [inline poll-taquito/src/components/Nav.tsx:13]: I would suggest not to add inline styles

**Comment by kylegrover** [comment]
I believe this PR was accidentally sent to this, you can close this here and I'll merge it over on my fork.

**Pull request closed without merging**